01-cloudformation/lab1-3/stack-mgmt.py

```diff
@@ -0,0 +1,82 @@
+from yaml.loader import SafeLoader
+import boto3
+import yaml
+import argparse
+import json
+
+## Usage python3 stack-mgmt.py --action delete --stack_name bgarlab13 --template_file template.yml --parameters_file parameters.json
+## Relies on relative path unless you specify the absolute path, didn't feel like implementing abspath :p
+
+parser = argparse.ArgumentParser(description='Allows for parameterization of execution')
```

Comment:
I would almost always isolate args in a way that allows the python code to be used in a module as well as from the command line.